Testscripts/Linux/containers_utils.sh

```diff
@@ -70,6 +70,7 @@ function VerifyDockerEngine() {
 function StartDockerEngine() {
     LogMsg "Start docker engine"
     systemctl start docker || service docker start
+    systemctl restart docker || service docker restart
```

Comment:
If always need restart, can it replace the above line 'systemctl start docker || service docker start' ?   or  should the logic be:  if the start failed, then try restart docker.

Reply:
@johnsongeorge-w is that okay? I fixed it since it returns non-zero code, when the service already running.

Reply:
changing from start->restart does not hurt.

Reply:
Like other PR, it could be better if you check the service status before direct the commands - start or restart.